projects/demo/src/app/app.routes.ts

```diff
@@ -11,10 +11,6 @@ export const routes: Routes = [
     path: 'components/accordion',
     loadComponent: () => import('./pages/accordion/accordion-demo.component').then((module) => module.AccordionDemoComponent),
   },
-  {
-    path: 'components/action-menu',
```

Comment:
if the action menu component is no longer needed, then it should be removed from `app.component.ts` (`menuConfigs`) and from the translation files as well